repose-aggregator/components/filters/rackspace-identity-basic-auth/src/main/scala/org/openrepose/filters/rackspaceidentitybasicauth/RackspaceIdentityBasicAuthHandler.scala

```diff
@@ -29,7 +31,7 @@ class RackspaceIdentityBasicAuthHandler(basicAuthConfig: RackspaceIdentityBasicA
   private val delegationWithQuality = Option(basicAuthConfig.getDelegating).map(_.getQuality)
   private val datastore = datastoreService.getDefaultDatastore
 
-  case class TokenCreationInfo(responseCode: Int, userId: Option[String], userName: String)
+  case class TokenCreationInfo(responseCode: Int, userId: Option[String], userName: String, retry: String)
```

Comment:
This allows us to propagate the Retry-After header's value if it is present. It goes along with the responseCode.